pydarn/plotting/superdarn_radars.py

```diff
@@ -1,101 +1,116 @@
-# Copyright (C) 2019 SuperDARN
-# Author: Marina Schmidt
+# Copyright (C) SuperDARN Canada, University of Saskatchewan
+# Authors: Marina Schmidt and Danno Peters
 
 """
 This module contains SuperDARN radar information
 """
 from typing import NamedTuple
 from enum import Enum
+from datetime import datetime
 
-
+"""
+_Hemispheres Class - Used to denote which hemisphere a radar is located in
+      _Hemisphere.North = 1
+      _Hemisphere.South = 0
+"""
 class _Hemisphere(Enum):
     North = 0
     South = 1
```

Comment:
Doesn't match docstring.  Also, I think it should be changed.

Reply:
done